src/aws/index_test.go

```diff
@@ -49,3 +55,46 @@ var4="val4"`
 		Expect(expectedTfString).To(Equal(actualSecrets.TfString()))
 	})
 })
+
+var _ = Describe("ECR helpers", func() {
+	It("compiles the list of image names", func() {
```

Comment:
Please put this in its own file `ecr_helpers_test.go` and add another `Describe` block for `GetImageNames` as that is what this it block applies to (not the entire ecr helpers)